src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -299,8 +304,11 @@ public String getVersion() {
      * @param stream Where to write the usage message.
      */
     public void usage(final PrintStream stream, final boolean printCommon) {
+
         if (prefix.isEmpty()) {
-            stream.print(getStandardUsagePreamble(callerOptions.getClass()) + getUsagePreamble());
+            final String prefix = htmlUnescape(convertFromHtml(getStandardUsagePreamble(callerOptions.getClass()) + getUsagePreamble()));
+            checkForNonASCII(prefix, "prefix");
```

Comment:
"prefix" is a weird way to refer to this part of the text. Prefix is generally understood as a short string prepended to some other string, like a name or identifier. If you really don't like "preamble" (which to be honest I'm not crazy about either, I was just using the existing nomenclature) then why not use something more descriptive like "tool description". Though for the variable name I would stick with "preamble" since that's how it's referred to in the rest of the code, for readability and consistency.